src/Engine/Rendering/Visualizers/OrbitVisualizer.cpp

```
@@ -0,0 +1,105 @@
+#include "OrbitVisualizer.hpp"
+
+
+OrbitVisualizer::OrbitVisualizer(const Ctx::VkRenderDevice *renderDeviceCtx, const Ctx::VkWindow *windowCtx, const Ctx::OffscreenPipeline *offscreenData, const Buffer::BufferAlloc *orbitVertBuffer, std::unordered_map<uint32_t, std::pair<size_t, uint32_t>> orbitVertexOffsets) :
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_alignedGlobalUBOSize, m_secondCmdBufs

(cppcoreguidelines-pro-type-member-init)